### ISSUE TITLE
Linreg info message shouldn't include intercept

### ISSUE THREAD

**Issue by tpoterba** [hail-is/hail#2777]
Running with no covariates, I get:
```
2018-01-22 07:28:22 Hail: INFO: Running linear regression for 1 phenotype on 284 samples with 1 covariate including intercept...
```

This is a bit confusing, I think. Maybe we could flip it so that it says `0 covariates (excluding intercept)`

**Comment by jbloom22** [comment]
how about "with intercept and 0 additional covariates..."?

**Comment by tpoterba** [comment]
that sounds good, yeah

**Comment by jbloom22** [comment]
How's this?
```
    info(s"Running linear regression on $n samples for ${ y.cols } response ${ plural(y.cols, "variables") } y,\n"
       + s"with input variables x, intercept, and $k additional ${ plural(k - 1, "covariate") }...")
```

**Comment by jbloom22** [comment]
Addressed by #2778
